lessons/231/python-app/Dockerfile

```diff
@@ -21,7 +21,7 @@ COPY --from=build /usr/local/bin /usr/local/bin
 COPY . /app
 
 # Match the number of CPU cores, which is 2 on m7a.large, for the 1st test to reduce throttling.
-# CMD ["fastapi", "run", "--workers", "2", "main.py"]
+# CMD ["uvicorn", "main:app", "--workers", "2", "--no-access-log"]
 
 # 4 for the second test to achieve full CPU utilization
-CMD ["fastapi", "run", "--workers", "4", "main.py"]
+CMD ["uvicorn", "main:app", "--workers", "4", "--no-access-log"]
```

Comment:
you should use gunicorn

Reply:
uvicorn is for development

Reply:
Maybe to state a "why". In some limited tests I did in the past, Gunicorn took ~75% of the RAM Uvicorn took and produced ~158% requests per second compared to the same app ran on Uvicorn.